wger/gym/templates/gym/inactive_gym_users_list.html

```diff
@@ -0,0 +1,385 @@
+{% extends "base.html" %}
+{% load i18n staticfiles wger_extras django_bootstrap_breadcrumbs %}
+
+{% block title %}{{gym}}{% endblock %}
+
+{% block breadcrumbs %}
+    {{ block.super }}
+
+    {% if perms.gym.manage_gyms %}
+        {% breadcrumb "Gyms" "gym:gym:list" %}
+        {% breadcrumb_raw gym "gym:gym:user-list" gym.pk %}
+        {% breadcrumb_raw name "gym:gym:inactive-user-list" gym.pk %}
+    {% endif %}
+{% endblock %}
+
+{% block content %}
+{% if perms.gym.manage_gym or perms.gym.gym_trainer %}
+
+<link rel="stylesheet" type="text/css" href="{% static 'bower_components/datatables/media/css/dataTables.bootstrap.min.css' %}">
+<script src="{% static 'bower_components/datatables/media/js/jquery.dataTables.min.js' %}" ></script>
+<script src="{% static 'bower_components/datatables/media/js/dataTables.bootstrap.min.js' %}" ></script>
+<script>
+$(document).ready( function () {
+    /* Make table sortable */
```

Comment:
***Code Quality: Critical***
This would quickly turn spaggetti I suggest you create an app.js files and have this go inside so that you would have one place where your javascript code lies.